held_suarez/held_suarez_1994.meta

```diff
@@ -65,57 +41,75 @@
   type = integer
   dimensions = ()
   intent = in
+[ pref_mid_norm ]
+  standard_name = reference_pressure_in_atmosphere_layer_normalized_by_reference_pressure
+  units = 1
+  type = real | kind = kind_phys
+  dimensions = (vertical_layer_dimension)
+  intent = in
 [ clat ]
   standard_name = latitude
   units = radians
   type = real | kind = kind_phys
   dimensions = (horizontal_loop_extent)
   intent = in
+[ cappa ]
+  standard_name = composition_dependent_ratio_of_dry_air_gas_constant_to_specific_heat_of_dry_air_at_constant_pressure
+  units = 1
+  dimensions = (horizontal_loop_extent, vertical_layer_dimension)
```

Comment:
Oops- maybe I was too hasty on my approval.  When looking at the ESCOMP/CAM PR, I looked back at this PR, and see that this does not have horizontal_loop_dimension for this run method.